src/test/jest.setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom';
```

Comment:
this is run as part of every test, effectively extending Jest's functionality with this. This allows asserting that things are in a DOM, etc